aws/logs_monitoring/steps/enrichment.py

```diff
@@ -97,6 +97,25 @@ def add_metadata_to_lambda_log(event):
     event[DD_CUSTOM_TAGS] = ",".join([event[DD_CUSTOM_TAGS]] + tags)
 
 
+def get_enriched_lambda_log_tags(log_event, cache_layer):
+    """Retrieves extra tags from lambda, either read from the function arn, or by fetching lambda tags from the function itself.
+
+    Args:
+        log (dict<str, str | dict | int>): a log parsed from the event in the split method
```

Comment:
is the best arg name log or log_event ?

Reply:
It will be a lambda log event here given the context. I think it should be good to use `log_event`

Reply:
OK, please just rename it in the comment then.